connectors/utils.py

```diff
@@ -770,7 +785,13 @@ async def extract_text(self, filepath, original_filename):
         )
 
         try:
-            content = await self.extract_with_file_send(filepath, filename)
+            async with self._begin_session().put(
```

Comment:
`self.filepointer_params(filepath)` I expect that this method always returns `filepointer_params` but not "". 

for the readability sake, I'd like to suggest to split this method into two instead of having 2 if-else statements.

Reply:
I separated them into two functions, it is definitely more readable now.